### PR TITLE
expose extractor via api scope

### PR DESCRIPTION
**Pull request by yausername**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).

### CODE REVIEW

**Comment by wb9688** [comment]
I think this change is good, however why are we currently able to call stuff in  `:extractor` from e.g. NewPipe then?

**Comment by yausername** [comment]
> I think this change is good, however why are we currently able to call stuff in `:extractor` from e.g. NewPipe then?

That's because jitpack packages a super module which exposes the root module and all submodules.
You can see the difference here (notice the `scope` tag in dependencies)
[the actual root module](https://jitpack.io/com/github/TeamNewPipe/NewPipeExtractor/NewPipeExtractor/v0.19.5/NewPipeExtractor-v0.19.5.pom)
[the super module packaged by jitpack](https://jitpack.io/com/github/TeamNewPipe/NewPipeExtractor/v0.19.5/NewPipeExtractor-v0.19.5.pom)

So in short, it works if you use jitpack.
